core/pouch/migrations.js

```diff
@@ -89,6 +89,30 @@ const migrations /*: Migration[] */ = [
         return doc
       })
     }
+  },
+  {
+    baseSchemaVersion: 2,
+    targetSchemaVersion: 3,
+    description: 'Marking Cozy Notes for refetch to avoid conflicts',
+    affectedDocs: (docs /*: Metadata[] */) /*: Metadata[] */ => {
+      return docs.filter(
+        doc =>
+          doc.mime === 'text/vnd.cozy.note+markdown' &&
```

Comment:
Shouldn't we check for `doc.metadata.content`?

Reply:
This might be a bit safer indeed.